lms/static/scripts/frontend_apps/common/test/dom-test.js

```diff
@@ -0,0 +1,60 @@
+import { listen } from '../dom';
+
+describe('sidebar/util/dom', () => {
```

Comment:
Can you update this to match the local path of the module relative to `frontend_apps` (ie. `common/dom`)